src/main/java/com/hamza/uber/entity/Driver.java

```diff
@@ -1,4 +1,21 @@
 package com.hamza.uber.entity;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
 public class Driver {
-}
+    // Prams -> driver, carModel, ratings, distanceFromCustomer
+    private String driver;
+    private String carModel;
+    private double rating;
+    private int distanceFromCustomer; // meters
```

Comment:
Avoid comments as much as possible. Your code should be descriptive enough so that you don't need comments to tell the meaning.

It's very hard to manage a code having lot of comments. your code will change with time and your comments the might become irrelevant or outdatted.

for example here, `checkMinDriverRating` can be called as `distanceFromCustomerInMeters`.

Reply:
True. I have updated the code with distanceFromCustomer -> distanceFromCustomerInMeters